ocp_resources/virtual_machine_export.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+
+
+from ocp_resources.resource import NamespacedResource
+from ocp_resources.virtual_machine import VirtualMachine
+from ocp_resources.virtual_machine_snapshot import VirtualMachineSnapshot
+
+
+class VirtualMachineExport(NamespacedResource):
+    """
+    VirtualMachineExport object.
+    """
+
+    api_group = NamespacedResource.ApiGroup.EXPORT_KUBEVIRT_IO
+
+    class SourceKind:
```

Comment:
Why this is needed here `class SourceKind:`?

Reply:
Because VirtualMachineExport can accept only 3 types of source objects: 
- VM
- VM Snapshot
- PVC

Reply:
Where this is used? In which cases do you use it?

Reply:
When we create a VirtualMachineExport object, like here:

```
    with cluster_resource(VirtualMachineExport)(
        name=f"vmexport-from-snapshot",
        namespace="default",
        client=admin_client,
        token_secret_ref="virt-export-token",
        source_api_group=VirtualMachineSnapshot.api_group,
        source_kind=VirtualMachineExport.SourceKind.VM_SNAPSHOT,
        source_name="my-vmsnapshot",
        teardown=False,
    ) as vm_export_from_snapshot:
```